browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -139,8 +144,13 @@ function handleAttachmentDrop (codeEditor, storageKey, noteKey, dropEvent) {
   const fileType = file['type']
 
   copyAttachment(filePath, storageKey, noteKey).then((fileName) => {
-    const showPreview = fileType.startsWith('image')
-    const imageMd = generateAttachmentMarkdown(originalFileName, path.join(STORAGE_FOLDER_PLACEHOLDER, noteKey, fileName), showPreview)
+    let previewType = null
+    if (fileType.startsWith('image')) {
+      previewType = 'image'
```

Comment:
maybe you should use constants istead of strings for this!

Reply:
Good idea!